Makefile

```diff
@@ -3,7 +3,7 @@ DIRS = $(filter-out _build/, $(dir $(wildcard */)))
 EXERCISES = $(patsubst %/, %, $(filter-out $(DIRS), $(dir $(wildcard */*/))))
 CATEGORIES = $(subst /, , $(DIRS))
 
-PROFILE ?= test $(EXERCISES) $(CATEGORIES)
```

Comment:
isn't this necessary to run a specific test?

Reply:
No, this is for rebar profiles. We have the default one `test` for when users run their solutions and `travis` for testing solutions (this is the one used by Travis)

Reply:
gotcha